app/main/forms/brief_forms.py

```diff
@@ -0,0 +1,91 @@
+from flask_wtf import Form
+from wtforms import IntegerField, SelectMultipleField
+from wtforms.validators import NumberRange
+
+
+class BriefSearchForm(Form):
```

Comment:
I don't really understand the `lot` problem to be honest. G-Cloud filters use content-loader to only show filters that apply to current lot, it feels like we should be able to easily ignore this if none of DOS filters specify lot dependencies.